catapult-sdk/src/plugins/accountRestrictions.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2016-present,
+ * Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp. All rights reserved.
+ *
+ * This file is part of Catapult.
+ *
+ * Catapult is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Catapult is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Catapult.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @module plugins/accountRestrictions */
+const EntityType = require('../model/EntityType');
+const ModelType = require('../model/ModelType');
+const sizes = require('../modelBinary/sizes');
+
+const constants = { sizes };
+
+const accountRestrictionTypeBlockOffset = 128;
+const accountRestrictionTypeFlags = Object.freeze({
+	address: 1,
+	mosaic: 2,
+	operation: 4
+});
+
+const accountRestrictionsCreateBaseCodec = valueCodec => ({
+	deserialize: parser => {
+		const transaction = {};
+		transaction.restrictionType = parser.uint8();
+		transaction.modifications = [];
+		const restrictionsCount = parser.uint8();
+		for (let i = 0; i < restrictionsCount; ++i) {
+			transaction.modifications.push({
+				type: parser.uint8(),
```

Comment:
why did you change this from modificationType?

Reply:
I have change it to "type" only because this is how it is in mongodb schema now.

Reply:
PR looks fine except for this doubt, please confirm ^^

Reply:
I see you changed to restrictionType in mongodb collection. I have updated.